packages/suite/src/hooks/wallet/coinmarket/form/useCoinmarketExchangeForm.ts

```diff
@@ -690,7 +694,8 @@ export const useCoinmarketExchangeForm = ({
         callInProgress,
         exchangeInfo,
         symbolsInfo,
-        quotes: filteredQuotes,
+        quotes: filteredCexQuotes,
```

Comment:
Here could be only  `quotes: filteredQuotes` more information in the upper comment.

Reply:
see the comment above :) I was event thinking adding here another collection - `comparatorQuotes` and move logic from `useFilteredQuotesByRateTypeAndExchangeType`. Quotes are already filtered so it's only about that one condition (`exchangeType === ExchangeType.DEX ? dexQuotes : filteredCexQuotes `).